code/index.html

```diff
@@ -3,28 +3,232 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>HIBR – It's like Uber, but for sleep.</title>
+    <link href="https://fonts.googleapis.com/css2?family=Lexend+Peta&family=Roboto+Mono:wght@400;700&family=Roboto+Slab:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <!-- Found a helpful guide which advocated it's ok to put the JS in the head, as long as you add the "defer" property: https://gomakethings.com/when-should-you-add-the-defer-attribute-to-the-script-element/#:~:text=In%20short%3A%20with%20defer%20on,downloaded%20and%20ready%20to%20run. -->
+    <script src="script.js" defer></script>
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+    <nav>
+      <div class="nav-content">
+        <div class="nav-logo"><img src="assets/hibr-logo.svg"></div>
+        <div class="nav-links">
+          <a href="#" class="nav-link">What's Hibr™?</a>
+          <a href="#" class="nav-link">Features</a>
+          <a href="#" class="nav-link">Hibr™ a friend</a>
+          <a href="#" class="nav-link">Legal</a>
+          <a href="#" class="nav-link">Guestbook</a>
+        </div>
+      </div>
 
-    <!-- Signup form -->
+    </nav>
+    
+    <header>  
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+      <section class="header-content">
```

Comment:
Happy to see you using rich dose of semantic tags, but over here it's not necessary - `div` is completely enough. `section`s best places for usage are main content parts, since over there will be a lot of them